### PR TITLE
[erlang-ls/erlang_ls#401] Add Erlang to Sublme LSP Client documentation

### PR DESCRIPTION
**Pull request by jfacorro**
Add Erlang to Sublme LSP Client documentation (... and also removes some trailing whitespaces).

Fixes erlang-ls/erlang_ls#401.

### CODE REVIEW

File: LSP.sublime-settings [hunk 1]
```diff
@@ -314,6 +314,13 @@
         "Packages/Elixir/Syntaxes/Elixir.tmLanguage",
       ]
     },
+    "erlang-ls": {
+      "command"   : [ "/path/to/my/erlang_ls", "--transport", "stdio" ],
+      "enabled"   : true,
```
Comment (rwols) [inline LSP.sublime-settings]: Please remove `"enabled": true` here.

Other than that, looks mergeable to me!

**Comment by jwortmann** [comment]
I just saw this PR. The double trailing whitespaces at the Fortran and Julia sections in the documentation were intentional, because 2 trailing spaces indicates a linebreak in Markdown. Now the correct formatting is gone. Also fenced code blocks aren't allowed inside lists, therefore the third item in the Erlang section wrongly has the number "1." again. https://lsp.readthedocs.io/en/latest/#erlang

**Comment by rchl** [comment]
Don't want to defend the change but relying on trailing whitespace sounds bad as many have settings to remove those automatically (or at least highlight as errors). Is there maybe a better way to handle the same (for example an extra blank line between items)?

**Comment by jfacorro** [comment]
@jwortmann Thank you for the heads up. PR addressing the issues mentioned [here](https://github.com/sublimelsp/LSP/pull/924).

**Pull request merged**